lib/inets/src/inets_app/inets.erl

```diff
@@ -434,10 +435,12 @@ report_event(Severity, Label, Service, Content) ->
 %%--------------------------------------------------------------------
 %%% Internal functions
 %%--------------------------------------------------------------------
-start_service(Service, Args, stand_alone) -> 
-    Service:start_standalone(Args);
-start_service(Service, Args, inets) ->
-    call_service(Service, start_service, Args).
+start_service(Service, Args, How) ->
```

Comment:
This is probably not a good idea, an in such case we probably should ensure inets is started too. Maybe this should be in the documentation that inets:start/[0,1]  should already been called (or applications started via release handling).
The whole inets module was designed to make more sense out of it once being such a strange mix of applications. Now the mix is only http client and server and which is less strange but still there is some relevance.